aibolit/patterns/non_final_attribute/non_final_attribute.py

```diff
@@ -0,0 +1,26 @@
+import javalang
+
+
+class NonFinalAttribute:
+
+    def __init__(self):
+        pass
+
+    def __file_to_ast(self, filename: str) -> javalang.ast.Node:
+        """
+        Takes path to java class file and returns AST Tree
+        :param filename:
+        :return: Tree
+        """
+        with open(filename, encoding='utf-8') as file:
+            tree = javalang.parse.parse(file.read())
+        return tree
+
+    def value(self, filename: str):
+        lst = []
+        tree = self.__file_to_ast(filename).filter(javalang.tree.FieldDeclaration)
+        for path, node in tree:
```

Comment:
@paulodamaso I think it is better to:
`return [node for node, _ in tree if 'final' not in node.modifiers]`